gcloud-java-pubsub/baseline/src/main/java/com/google/cloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -306,7 +324,7 @@ private Topic createTopic(Topic request) {
 
   // AUTO-GENERATED DOCUMENTATION AND METHOD - see instructions at the top of the file for editing.
   /**
-   * Adds one or more messages to the topic. Generates `NOT_FOUND` if the topic
+   * Adds one or more messages to the topic. Returns `NOT_FOUND` if the topic
```

Comment:
It looks like we are losing a documentation update that was made for pubsub. (We updated the wording of "Returns" and changed it to "Generates")